### PR TITLE
WIP POC Jaeger Embedded components

### PR DESCRIPTION
**Pull request by aljesusg**
Just a Poc

From the PoC embed components in jaeger https://github.com/jaegertracing/jaeger-ui/pull/263

### Distribute tracing link

We should have here or own search for the traces
resukts

### Traces of a service 
resutls2

### On trace
trace_vire

### Video Show
poc kiali

cc @lucasponce

### CODE REVIEW

**Comment by rhqci** [comment]
:heavy_check_mark: Jenkins CI: kiali-ui-pr-e2e-test [#180](https://paste.fedoraproject.org/paste/VD7FVM9ZUgTSsojOHlEb-g)
  * :heavy_check_mark: Istio-and-Kiali-on-Openshift [#[1530]](https://paste.fedoraproject.org/paste/8APKzxfcdhifcigHaqbzLQ)
  * :heavy_check_mark: Istio-Upstream [#[388]](https://paste.fedoraproject.org/paste/Mcbms4zgBw7eBMHWlXA5EQ)
  * :heavy_check_mark: Kiali-Istio-Checker [#[1611, 1612]](https://paste.fedoraproject.org/paste/2Q7ir6VjVpjNK9RmPP74SQ)
  * :heavy_check_mark: install-bookinfo [#[900]](https://paste.fedoraproject.org/paste/5d31VszDDWynSjYMk7JzZg)
  * :heavy_check_mark: Kiali-on-Openshift [#[904]](https://paste.fedoraproject.org/paste/kVoJfQMtKjUEkbFA378u0A)
  * :heavy_check_mark: run-kiali-e2e-tests [#[328]](https://paste.fedoraproject.org/paste/t4zCt9AAY-zttNkDl3hvgQ)


KIALI_NAME=kialiuipr736-istio-system

File: src/pages/ServiceJaeger/ServiceJaegerPage.tsx [hunk 1]
```diff
@@ -28,10 +28,14 @@ const EmptyStatePage = () => (
 class ServiceJaegerPage extends React.Component<RouteComponentProps<{}>, ServiceJaegerState> {
   constructor(props: RouteComponentProps<{}>) {
     super(props);
-    this.state = { jaegerURL: '', error: false };
+    this.state = {
```
Comment (mtho11) [inline src/pages/ServiceJaeger/ServiceJaegerPage.tsx]: Is the jaegerURL really stateful? Do we switch jaegerURLs in the application and expect a render?

Comment (aljesusg) [reply src/pages/ServiceJaeger/ServiceJaegerPage.tsx]: No @mtho11, this will be like before we request to the backend for the Jaeger url, this is a PoC

**Comment by rhqci** [comment]
:heavy_check_mark: Jenkins CI: kiali-ui-pr-e2e-test [#189](https://paste.fedoraproject.org/paste/dAtJ6CdRC9FhzadSSWZ44g)
  * :heavy_check_mark: Istio-and-Kiali-on-Openshift [#[1548]](https://paste.fedoraproject.org/paste/Wt5xJ1gHmbiyY798s46Rug)
  * :heavy_check_mark: Istio-Upstream [#[405]](https://paste.fedoraproject.org/paste/ytyz3jVOZwLQHIVWLn6kkA)
  * :heavy_check_mark: Kiali-Istio-Checker [#[1650, 1651]](https://paste.fedoraproject.org/paste/ZmvJkxsWJEOYQ2nTYX8BFQ)
  * :heavy_check_mark: install-bookinfo [#[918]](https://paste.fedoraproject.org/paste/ux1mSvl6z7b66HhNcdxeMA)
  * :heavy_check_mark: Kiali-on-Openshift [#[925]](https://paste.fedoraproject.org/paste/2KxcAsr9WmL4Wy7xNMwqBg)
  * :heavy_check_mark: run-kiali-e2e-tests [#[346]](https://paste.fedoraproject.org/paste/CU7UJ2YEuyTkoS3UGIlRXw)


KIALI_NAME=kialiuipr736-istio-system

**Comment by clyang82** [comment]
@aljesusg I saw there has a ticket to move iframe to external links - https://github.com/kiali/kiali-ui/pull/619. Right now, it seems you are working on using iframe to embed the jaeger again. What is the reason to use iframe? Thanks.

**Comment by lucasponce** [comment]
Hi @clyang82,

The main JIRA explaining the brackground of this work is under this link https://issues.jboss.org/browse/KIALI-1406.

Basically Kiali's goal is to offer added value integration with Jaeger.
In our first step we integrated in the app as an iFrame, but it didn't offer high integration, only a minimal filtering, also it had some issues working with an embedded proxy.

That was the main motivation to move Jaeger as an external link until we have progress in a consistent integration.

@aljesusg is working with the Jaeger community to allow to consume the UI in a better component driven https://github.com/jaegertracing/jaeger-ui/issues/248 then we can integrate it with Kiali sharing filtering as same time interval used for collect metrics and traces.

**Comment by lucasponce** [comment]
@clyang82 yes, if you follow the Jaeger UI discussion, main developers want to follow a similiar Google API approach (talking about the implementation of this approach) but the idea behind it is to reuse Jaeger in a standard way (so not discarded that an iframe is part of the implementation after all).
Also our plans are to collaborate more closely with the Jaeger community, so any changes proposed will be consolidated in a standard way.

**Comment by clyang82** [comment]
@lucasponce I vote this enhancement. What is the plan to deliver this feature?

**Comment by lucasponce** [comment]
@clyang82  on this Sprint 12, @aljesusg is mainly focus to research and be in contact with the Jaeger community about this feature.
There are some PRs to start moving Jaeger UI as a component, once those are consolidated, in following sprints we will propagate these changes in Kiali.
Also there are some multitenancy/security aspects related that are WIP, but Jaeger integration is top priority for Kiali, so count we will be working on that direction on this and next sprints.

**Pull request closed without merging**